packages/framework/src/Model/Category/Category.php

```diff
@@ -210,11 +214,114 @@ protected function setTranslations(CategoryData $categoryData)
         }
     }
 
+    /**
+     * @param int $domainId
+     * @return string|null
+     */
+    public function getSeoTitle($domainId)
+    {
+        return $this->getCategoryDomain($domainId)->getSeoTitle();
+    }
+
+    /**
+     * @param int $domainId
+     * @return string|null
+     */
+    public function getSeoH1($domainId)
+    {
+        return $this->getCategoryDomain($domainId)->getSeoTitle();
+    }
+
+    /**
+     * @param int $domainId
+     * @return bool
+     */
+    public function isEnabled($domainId)
+    {
+        return $this->getCategoryDomain($domainId)->isEnabled();
+    }
+
+    /**
+     * @param $domainId
+     * @return bool
+     */
+    public function isVisible($domainId)
+    {
+        return $this->getCategoryDomain($domainId)->isVisible();
+    }
+
+    /**
+     * @param int $domainId
+     * @return string|null
+     */
+    public function getSeoMetaDescription($domainId)
+    {
+        return $this->getCategoryDomain($domainId)->getSeoMetaDescription();
+    }
+
+    /**
+     * @param int $domainId
+     * @return string|null
+     */
+    public function getDescription($domainId)
+    {
+        return $this->getCategoryDomain($domainId)->getDescription();
+    }
+
+    /**
+     * @return \Doctrine\Common\Collections\ArrayCollection|\Shopsys\FrameworkBundle\Model\Category\CategoryDomain[]
+     */
+    public function getCategoryDomains()
```

Comment:
In Brand, this method wasn't public.

Reply:
it was. as i said, i think that there is not other solution to make this method private